src/Microsoft.CodeAnalysis.Razor.Workspaces/AcceptedCharacters.cs

```diff
@@ -3,7 +3,7 @@
 
 using System;
 
-namespace Microsoft.VisualStudio.LanguageServices.Razor
+namespace Microsoft.CodeAnalysis.Razor
```

Comment:
Your strategy assumes/requires that we don't make any changes to the services as they are currently specified. I'm not sure that's the case.